### ISSUE TITLE
Support for ephemeral runner

### ISSUE THREAD

**Issue by kare** [MonolithProjects/ansible-github_actions_runner#110]
Any plans to support ephemeral runners?

https://github.blog/changelog/2021-09-20-github-actions-ephemeral-self-hosted-runners-new-webhooks-for-auto-scaling/

I think ephemeral should be the default.

**Comment by MonolithProjects** [comment]
Hey @kare thanks for this issue. I don't think this should be a default option. Anyway, you can set the runner as ephemeral even now using `runner_extra_config_args` variable:

```yaml
---
- name: Install GitHub Actions Runner
  hosts: all
  user: ansible
  become: yes
  vars:
    - github_account: github-access-user
    - github_repo: my_awesome_repo
    - runner_extra_config_args: "--ephemeral"
  roles:
    - role: monolithprojects.github_actions_runner
```

Btw i just found out that `runner_extra_config_args` has not been working for Organization Self-hosted Runners. This is now fixed in release 1.12.2.

**Comment by kare** [comment]
Thanks!
